frontend/routes/admin/tickets.tsx

```diff
@@ -12,6 +12,9 @@ export default define.page<typeof handler>(function Tickets({
   data,
   url,
 }) {
+  const isNotification = (ticket: Ticket) =>
```

Comment:
instead of this function, create a variable in the tickets.map call

Reply:
good idea